src/runtime-rs/crates/runtimes/wasm_container/.gitignore

```diff
@@ -0,0 +1,10 @@
+.tmp
+/target
+**/*.rs.bk
+.vscode
+*.pyc
+.idea
+Cargo.lock
+src/*.rs.swp
+src/*/*.rs.swp
```

Comment:
We can have a top level `.gitinore` (e.g. under `src/runtime-rs`) so that the same file does not have to repeat itself everywhere.

Reply:
Thanks for your review, already delete .gitignore.